src/js/app.js

```
@@ -653,64 +653,38 @@ function scrollToFocused() {
-
-function handleNavigationDown(e, n) {
-  e.preventDefault();
-  const cols = GRID_COLS();
-  focusedIdx = Math.min(focusedIdx + cols, n - 1);
-  if (focusedIdx < 0) focusedIdx = 0;
-  if (focusedIdx >= visibleCount) {
-    visibleCount = Math.min(visibleCount + 12, n);
-    renderOrgs(false);
+function openCompare(){
+  renderCompareSlots();
+  renderCompareTable();
```

ESLint

[error] 'renderCompareTable' is not defined.

(no-undef)